Appprogrammering/app/src/main/java/com/example/appprogrammering2/MainActivity.kt

```diff
@@ -1,23 +1,39 @@
 package com.example.appprogrammering2
 
+import android.app.Activity
+import android.content.Intent
 import android.graphics.Color
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.View
-import android.widget.Button
-import android.widget.TextView
+import android.widget.*
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
+        val intent = Intent(this, secondActivity::class.java)
+        val activityButton = findViewById<Button>(R.id.activityButton)
         val inputButton = findViewById<Button>(R.id.sendButton)
         val darkButton = findViewById<Button>(R.id.button2)
         val background = findViewById<View>(R.id.frontPage)
         val inputText = findViewById<View>(R.id.inputText) as TextView
         val textField = findViewById<View>(R.id.textField) as TextView
 
+        val spinner: Spinner = findViewById(R.id.color_spinner)
+        // Create an ArrayAdapter using the string array and a default spinner layout
+        ArrayAdapter.createFromResource(
+            this,
+            R.array.color_array,
+            android.R.layout.simple_spinner_item
+        ).also { adapter ->
+            // Specify the layout to use when the list of choices appears
+            adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
+            // Apply the adapter to the spinner
+            spinner.adapter = adapter
+        }
+
```

Comment:
Det her ligner altså ret meget copy paste.

Reply:
Regner du med jeg kan finde ud af at programmere android studio selv?

Reply:
Jeg regner med at du kan vise at du kan programmere... ikke kopiere andres løsning. 

Når du afleverer kode som er tydeligt copy pastet kan jeg jo blive bekymret for om du faktisk har lært det. Kom forbi efter pausen hvis du gerne vil diskutere det yderligere.